proto/storage/alert.proto

```diff
@@ -17,11 +17,11 @@ message Alert {
         string                  id                  = 1 [(gogoproto.moretags) = "search:\"Deployment ID,store,hidden\" sql:\"index=hash\""];
         string                  name                = 2 [(gogoproto.moretags) = "search:\"Deployment,store\""];
         string                  type                = 4;
-        string                  namespace           = 5 [(gogoproto.moretags) = 'search:"Namespace"'];
-        string                  namespace_id        = 16 [(gogoproto.moretags) = 'search:"Namespace ID,hidden"'];
+        string                  namespace           = 5;  // This field has to be duplicated in Alert for scope management and search.
+        string                  namespace_id        = 16; // This field has to be duplicated in Alert for scope management and search.
         map<string, string>     labels              = 7;
-        string                  cluster_id          = 9  [(gogoproto.moretags) = 'search:"Cluster ID,hidden"'];
-        string                  cluster_name        = 10 [(gogoproto.moretags) = 'search:"Cluster,hidden"'];
+        string                  cluster_id          = 9;  // This field has to be duplicated in Alert for scope management and search.
+        string                  cluster_name        = 10; // This field has to be duplicated in Alert for scope management and search.
```

Comment:
If you remove search tag how it could be used for search?

Reply:
Alerts exist in three flavours:
- Deployment alerts
- Image alerts
- Resource alerts
For deployment and resource, it's easy to find in which kubernetes cluster and namespace the alert fired. For image alerts, it's a bit more complicated, and currently not really checked.
The search tags being removed here are the ones for deployment alerts.
`Cluster ID` and `Namespace` search fields are now duplicated at the root of the `Alert` proto message when upserting. These fields are located lines 108 to 111 in the current file. The flags are being removed here to reduce the chances of collision on search fields.

Reply:
Yeah this seems fine